### DISCUSSION TITLE
Unterschied zu Gitlab?

### DISCUSSION THREAD

**Discussion by Torodde** [wulfland/BootCampBasta23#7] [General]
Was ist der großer Unterschied bzw. Vorteil von GitHub gegenüber [GitLab](https://about.gitlab.com/de-de/)? :innocent:

**Comment by wulfland** [comment]
Ist das bessere Tool 😎

**Reply by Gorphi** [reply]
@wulfland : Wenn du slides hast die beide Systeme vergleichen, würden mich diese auch interssieren...
